core/pom.xml

```diff
@@ -87,11 +92,6 @@
 
         <!-- dependencies shaded by us-->
 
-        <dependency>
-            <groupId>za.co.absa.shaded</groupId>
-            <artifactId>absa-shaded-jackson</artifactId>
-            <version>0.0.1</version>
-        </dependency>
```

Comment:
I felt like we should declare dependencies that we use directly, even though they are used transitively as well. Do you not agree?